babel.config.js

```diff
@@ -0,0 +1,19 @@
+const presets = [
+  [
+    "@babel/env",
+    {
+      targets: {
```

Comment:
Is it worth adding babel at a later stage to this work? I understand adding it to the config is easy, but we have to consider how & when we'll apply transpiling (see considerations in https://github.com/mozilla/bedrock/issues/7542).